source/documentation/transaction.html.md

```diff
@@ -133,11 +133,14 @@ val fResult = DB localTx { implicit s =>
 ```
 
 <hr/>
-### #Working with IO monads
```

Comment:
I wanted you to change only this section.

Reply:
Reverted the titles except this one.